src/xenomods/State.cpp

```diff
@@ -88,24 +89,25 @@ namespace xenomods {
 						return false;
 					}
 				});
-			}
-			else
+			} else
 				load_failed = true;
 
 			if(load_failed && respectDefaults)
 				bdatSkipOverrides = CONFIG_BDAT_SKIP_OVERRIDES_DEFAULT;
 		}
 
-		if (respectDefaults || table[STRINGIFY(mountTornaContent)].type() != toml::node_type::none)
+		if(respectDefaults || table[STRINGIFY(mountTornaContent)].type() != toml::node_type::none)
 			mountTornaContent = table[STRINGIFY(mountTornaContent)].value_or(CONFIG_MOUNT_TORNA_CONTENT_DEFAULT);
 
-		if (respectDefaults || table[STRINGIFY(enable60FPS)].type() != toml::node_type::none)
+		if(respectDefaults || table[STRINGIFY(enable60FPS)].type() != toml::node_type::none)
 			enable60FPS = table[STRINGIFY(enable60FPS)].value_or(CONFIG_ENABLE_60FPS_DEFAULT);
 
-		if (respectDefaults && table[XENOMODS_CODENAME_STR].is_table()) {
+		if(respectDefaults && table[XENOMODS_CODENAME_STR].is_table()) {
 			//g_Logger->LogDebug("Found {} as a category, loading...", XENOMODS_CODENAME_STR);
 			InitializeFromTable(*table[XENOMODS_CODENAME_STR].as_table(), false);
 		}
+		if(respectDefaults || table[STRINGIFY(loadFcLatest)].type() != toml::node_type::none)
```

Comment:
This should come just before it loads the codename table, right after the enable60FPS check